problems/removeEvenStrings.js

```diff
@@ -6,8 +6,9 @@
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
 
-function removeEvenStrings() {
-
+function removeEvenStrings(arr) {
+    let arr = 'hello';
+    return arr.filter(num => num % 2)
 }
 
```

Comment:
Filter is the right method, but we're checking the wrong thing. It's an array of strings, so we need to check the str.length inside fo the callback.